apps/studio/lib/ai/model.ts

```diff
@@ -14,8 +14,9 @@ export const regionMap = {
   eu: 'eu',
 }
 
-const SONNET_MODEL = 'anthropic.claude-3-7-sonnet-20250219-v1:0'
-const HAIKU_MODEL = 'anthropic.claude-3-haiku-20240307-v1:0'
+const IS_THROTTLED = process.env.IS_THROTTLED !== 'false'
```

Comment:
to make it more clear hahaha

```suggestion
const IS_THROTTLED = process.env.IS_THROTTLED === 'true'
```

Reply:
@joshenlim this was intentional as a safety precaution so that it defaults to false always

Reply:
default behaviour should be throttled

Reply:
this is really just a temporary solution. throttling should by dynamic and we have the isThrottled prop for this, its just not in use right now as we experiment with different models